src/Destination/S3CmdDestination.php

```diff
@@ -112,7 +113,7 @@ public function all()
             throw new \RuntimeException($process->getErrorOutput());
         }
 
-        return $this->parseS3CmdListOutput($process->getOutput());
+        return new BackupCollection($this->parseS3CmdListOutput($process->getOutput()));
```

Comment:
The only reason it is there is for BC. I still use it on some projects. I definitely want to remove it later - perhaps like you say, create a hard dependency on Flysystem and remove StreamDestination as well.

I'm concerned about the efficiency of Flysystem for sending large files compared to s3cmd. Have any experience with that?